src/screens/TopicMemberScreen/index.js

```diff
@@ -154,7 +154,6 @@ const TopicMemberScreen = () => {
     cancelTokenRef.current = axios.CancelToken.source();
   };
 
-  if (isInitialLoading) return null;
   return (
     <SafeAreaProvider forceInset={{top: 'always'}} style={styles.parentContainer}>
       <NavHeader
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The condition `if (isInitialLoading) return null;` has been removed. This could potentially lead to undesired behavior if the component tries to render while it's still in the initial loading state. If you want to prevent rendering during the initial loading state, consider adding the condition back.
```diff
+  if (isInitialLoading) return null;
   return (
     <SafeAreaProvider forceInset={{top: 'always'}} style={styles.parentContainer}>
       <NavHeader
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->